app/retail/templates/home/index2020.html

```diff
@@ -121,26 +123,40 @@ <h1>
   <!-- what is gitcoin -->
 
   <section id="whatisgitcoin">
-    <div class="grid">
-      <div>
-        <h2>Here to earn?</h2>
-        <p>
-          Build exciting projects and upskill on development while earning
-          Ethereum and other tokens. Find mentorship by connecting with
-          talented developers and leaders of Ethereum’s top companies.
-        </p>
-        <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
-        <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
-        <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+    <div class="container">
```

Comment:
Adding this container wrapper won't make this block align with the rest of the page.

Reply:
did you try running this locally? @aamustapha

Reply:
why not just use row? 
Isn't the entire page wrapped within a container already ?

Reply:
@thelostone-mc no it isn't. i introduced bootstrap... didn't want to mess with the existing classes

Reply:
I don't have to run it. .container would set a max width for the div.

Reply:
argh :/ mannn this is painful 
cool cool makes sense